mobile/src/people/employees-list-item.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { TouchableHighlight, StyleSheet, Platform, Text, View, Image } from 'react-native';
+
+import { Avatar } from '../people/avatar';
+import { Employee } from '../reducers/organization/employee.model';
+import { employeesListItemStyles as styles } from './styles';
+import { StyledText } from '../override/styled-text';
+
+interface EmployeesListItemProps {
+    employee: Employee;
+    id: string;
+}
+
+export class EmployeesListItem extends React.Component<EmployeesListItemProps> {
+    public render() {
+        const employeeName = this.props.employee ? this.props.employee.name : 'Unknown';
+        const employeePosition = this.props.employee ? this.props.employee.position : 'Unknown';
+        const photo = this.props.employee ? this.props.employee.photo : null;
+        const mimeType = photo ? photo.mimeType : null;
+        const base64 = photo ? photo.base64 : null;
+
+        return (
+            <TouchableHighlight>
+                <View style={styles.layout}>
+                    <View style={styles.avatarContainer}>
+                        <Avatar mimeType={mimeType} photoBase64={base64} style={StyleSheet.flatten(styles.avatar)} />
```

Comment:
does it even compile? Avatar takes photo now.

Reply:
compile but no real photo, just stub, I was wondering why - will be fixed :)